_data/cloudservices.yml

```diff
@@ -4692,11 +4686,11 @@ services:
             ref: https://cloud.google.com/solutions/iot/
             icon: Google-Cloud-Platform.png
       - ibm:
-          - name: Internet of Things Platform
+          - name: Internet of Things Platform (deprecated)
             ref: https://cloud.ibm.com/docs/services/IoT/getting-started.html
```

Comment:
your ref is broken,  now it is 

https://cloud.ibm.com/catalog/services/internet-of-things-platform